### DISCUSSION TITLE
Is there a way to import ALL my watch history from Trakt to my plex database (even for items that are deleted from Plex)?

### DISCUSSION THREAD

**Discussion by RileyXX** [Taxel/PlexTraktSync#1293] [General]
So I know this script can import watch history from Trakt into Plex and vise versa. But from what I can tell it only imports watch history for items that are currently available to watch in my Plex library. The thing is I want to add ALL my watch history for items dating back several years into my Plex library, including items that are no longer available to watch in my Plex library. That way when I add old content to my Plex library the watch history is already available. Is this possible with this script?

I currently have the script set to auto run and import my watch history from plex to trakt. But I have trakt to plex disabled because sometimes things get marked as played by accident. And I have all my shows set to auto delete watched episodes. So in my case it's not worth it to auto import from trakt to plex since things will randomly get deleted on occasion. So I'm trying to find a way to import my old watch history all at once that way I don't need to have trakt to plex sync enabled.

**Comment by simonc56** [comment]
- #1142

**Comment by glensc** [comment]
closing in favor of feature request
